csharp/HOLMS.Platform/HOLMS.Platform/Types/Topics/BookingContextTopics.cs

```diff
@@ -4,6 +4,7 @@ public class BookingContextTopics {
 
         public const string ReservationCancellation = "bookings.reservations.cancellations";
         public const string ReservationCheckInTopic = "booking.reservations.checkins";
+        public const string ReservationCheckOutTopic = "booking.reservations.checkouts";
```

Comment:
Man, I need to be carrying dust/sentries. These invisible lines of code are killing me.